_posts/2020-01-20-workshop.md

```diff
@@ -18,22 +18,22 @@ clinicians from around the world and discuss together and establish new collabor
 
 ### Program:
 
-| 1st Day | Monday, 20st January 2020<br> [IoE Bedford Way (20) - Different room depending the session](https://20bedfordway.com/how-to-find-us/)  |
+| 1st Day | Monday, 20th January 2020<br> [IoE Bedford Way (20) - Different room depending the session](https://20bedfordway.com/how-to-find-us/)  |
 |---|---|
-| 13:30 - 14.00| <b>Welcome session - Room 642 breakdown 1 (6th floor)</b><br>Daniel Alexander (Director of Centre for Medical Image Computing)<br><i>The Centre for Medical Image Computing</i><br><br>Ferran Prados (Centre for Medical Image Computing - UCL - UK)<br><i>Welcome to the Spinal Cord MRI workshop</i> |
-| 14:00 - 14.30 | Coffee break |
-| 14:30 - 16.30 | <b>Session 1 - Room 642 breakdown 1 (6th floor)</b><br>Julien Cohen-Adad (Polytechnique Montreal - Canada)<br><i>TBC</i><br><br>Hugo Vrenken (VUMC - Amsterdam - Netherland)<br><i>TBC</i><br><br>Jalesh Paniker (Institute of Neurology - UCL - UK)<br><i>TBC</i><br><br>Mark Horsfield (Xinapse Systems - UK )<br><i>TBC</i><br><br> |
-| 16:30 - 17.00 | Coffee break |
-| 17:00 - 18:30 | <b>Session 2 - C3.09 Lecture Theatre (3rd floor)</b><br> John Ashburner (Institute of Neurology - UCL - UK)<br><i>TBC</i><br><br> Torben Schneider (Philips UK)<br><i>TBC</i><br><br> Jon Stutters (Queen Square Multiple Sclerosis Centre - UCL - UK)<br><i>TBC</i><br><br> |
+| 13:30 - 14:00| <b>Welcome session - Room 642 breakdown 1 (6th floor)</b><br>Daniel Alexander (Director of Centre for Medical Image Computing)<br><i>The Centre for Medical Image Computing</i><br><br>Ferran Prados (Centre for Medical Image Computing - UCL - UK)<br><i>Welcome to the Spinal Cord MRI workshop</i><br><br>Emma Gray (MS Society - UK)<br><i>The value of spinal cord imaging for MS Society UK</i><br>|
+| 14:00 - 14:30 | Coffee break |
+| 14:30 - 16:30 | <b>Session 1 - Room 642 breakdown 1 (6th floor)</b><br>Julien Cohen-Adad (Polytechnique Montreal - Canada)<br><i>TBC</i><br><br>Jalesh N. Paniker (Institute of Neurology - UCL - UK)<br><i>The lumbosacral spinal cord</i><br><br>Mark Horsfield (Xinapse Systems - UK )<br><i>Spinal cord atrophy assessment using JIM</i><br><br>Hugo Vrenken (VUMC - Amsterdam - Netherland)<br><i>TBC</i><br><br> |
```

Comment:
title for my talk:
Standardizing acquisition and processing of spinal cord MRI data